tool/builder.dart

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2018, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:convert';
+
+import 'package:build/build.dart';
+import 'package:glob/glob.dart';
+
+Builder benchmarkBuilder(BuilderOptions options) => _BenchmarkBuilder();
+
+class _BenchmarkBuilder implements Builder {
+  @override
+  Future build(BuildStep buildStep) async {
+    var data = <String, String>{};
+
+    await for (var item in buildStep
+        .findAssets(Glob('benchmark/**/*.pb.json'))
+        .where((id) =>
+            id.pathSegments.length > 2 &&
+            id.pathSegments[0] == 'benchmark' &&
```

Comment:
Is this line needed?

Reply:
Strictly, no. But I think it helps folks understand what's going on...